taskcluster/translations_taskgraph/transforms/cached_tasks.py

```diff
@@ -82,6 +84,9 @@ def add_cache(config, jobs):
                             digest_data.append(f"{param}:{value}")
                             break
 
+        if cache_version:
```

Comment:
Ok, the changes here are quite small. I see you're keeping the files and just adding another way to invalidate caches. Up you you if you want to separate this.